core/app/modules/kit-library/assets/js/components/item-header.js

```diff
@@ -0,0 +1,145 @@
+import { useNavigate } from '@reach/router';
+import { Dialog, Text } from '@elementor/app-ui';
+import { useSettingsContext } from '../context/settings-context';
+import Header from './layout/header';
+import HeaderBackButton from './layout/header-back-button';
+import HeaderButtons from './layout/header-buttons';
+import Kit from '../models/kit';
+import ConnectDialog from './connect-dialog';
+
+const { useMemo, useState } = React;
+
+import './item-header.scss';
+
+export default function ItemHeader( props ) {
+	const navigate = useNavigate();
+	const { updateSettings } = useSettingsContext();
+
+	const [ isConnectDialogOpen, setIsConnectDialogOpen ] = useState( false );
+	const [ isImportDialogOpen, setIsImportDialogOpen ] = useState( false );
+	const [ error, setError ] = useState( false );
+
+	const applyButton = useApplyButton( props.model, {
+		onConnect: () => setIsConnectDialogOpen( true ),
+		onApply: () => setIsImportDialogOpen( true ),
+	} );
+
+	const buttons = useMemo( () => [ applyButton, ...props.buttons ], [ props.buttons, applyButton ] );
+
+	return (
+		<>
+			{
+				error && <Dialog
+					title={ error }
+					approveButtonText={ __( 'Learn More', 'elementor-pro' ) }
+					approveButtonColor="link"
+					approveButtonUrl="#"
+					approveButtonOnClick={ () => setError( false ) }
+					dismissButtonText={ __( 'Close', 'elementor-pro' ) }
+					dismissButtonOnClick={ () => setError( false ) }
+					onClose={ () => setError( false ) }
+				/>
+			}
+			{
+				isConnectDialogOpen && <ConnectDialog
+					onClose={ () => setIsConnectDialogOpen( false ) }
+					onSuccess={ ( data ) => {
+						updateSettings( {
+							is_library_connected: true,
+							access_level: data.access_level || 0,
+						} );
+
+						if ( data.access_level < props.model.accessLevel ) {
+							return;
+						}
+
+						setIsImportDialogOpen( true );
+					} }
+					onError={ ( message ) => setError( message ) }
+				/>
+			}
+			{
+				isImportDialogOpen && <Dialog
+					title={ __( 'Apply %s?', 'elementor' ).replace( '%s', props.model.title ) }
+					approveButtonText={ __( 'Apply All', 'elementor' ) }
+					approveButtonOnClick={ () => {
+						navigate( '/import' );
```

Comment:
Replace the popup with a button that navigates to `/import/process?file-url=https...&nonce=bhjvhj`